.github/workflows/simulation.yml

```diff
@@ -203,6 +203,37 @@ jobs:
     - name: Codecov
       uses: codecov/codecov-action@v1
 
+  external-commissioner:
+    continue-on-error: true
```

Comment:
I think we should fail on error. 
We **should** fix the issue if code change of the PR cause `external-commissioner` tests to fail.

Reply:
@bukepo @wgtdkp Should we fail on error? Any incompatible changes made to `ot-commissioner` test scripts may make this fail, is such dependencies OK?

Reply:
I'm fine with either way.

Reply:
Removed continue-on-error.

Reply:
I am only fine with `continue-on-error: false`